tests/bsim/bluetooth/host/gatt/notify_multiple/test_scripts/_notify-debug.sh

```diff
@@ -15,6 +15,7 @@ source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
 
 simulation_id="notify_multiple"
 verbosity_level=2
+BOARD="${BOARD:-nrf52_bsim/native}"
```

Comment:
This line was just removed in a previous commit. Seems this is a bad rebase merge(?)